### PR TITLE
docs: HTTP support

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: src/pages/http.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+export const description = 'Integrating with existing HTTP frameworks'
+
+# HTTP
+
+Nitric supports existing web app and API frameworks. The `http` resource allows you to connect nitric to your existing applications.
+
+## Enabling in your project
+
+HTTP framework support is currently in preview, you enable it in your nitric project add the following to your `nitric.yaml` file
```
Comment (HomelessDinosaur) [inline src/pages/http.mdx:9]:
```suggestion
HTTP framework support is currently in preview, to enable it in your Nitric project add the following to your `nitric.yaml` file
```

File: src/pages/http.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+```yaml
+preview-features:
+  - http
+```
+
+## Connecting existing frameworks
+
+Here are some examples of how to connect your existing API frameworks with nitric:
```
Comment (HomelessDinosaur) [inline src/pages/http.mdx:18]:
```suggestion
Here are some examples of how to connect your existing API frameworks with Nitric:
```

File: src/pages/http.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+export const description = 'Integrating with existing HTTP frameworks'
+
+# HTTP
+
+Nitric supports existing web app and API frameworks. The `http` resource allows you to connect nitric to your existing applications.
```
Comment (HomelessDinosaur) [inline src/pages/http.mdx:5]:
```suggestion
Nitric supports existing web app and API frameworks. The `http` resource allows you to connect Nitric to your existing applications.
```

File: src/pages/http.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+async function bootstrap(port: number) {
+  const app = await NestFactory.create(AppModule)
+  await app.listen(port)
+}
+
+http(bootstrap)
+```
+
+```python
```
Comment (HomelessDinosaur) [inline src/pages/http.mdx:48]: Maybe this should be a top level <Note> instead of a comment?

**Comment by davemooreuws** [comment]
Is it worth adding to building blocks in home mdx. Or replacing a less popular one with it.

File: src/pages/http.mdx (hunk 4 above)
Comment (tjholm) [reply src/pages/http.mdx:48]: wrote the same way as custom domain support for other clouds, I think the intention was to convey we're planning support soonish

**Comment by tjholm** [comment]
> Is it worth adding to building blocks in home mdx. Or replacing a less popular one with it.

Maybe once its out of preview

**Review by HomelessDinosaur — APPROVED** [review]

**Review by davemooreuws — DISMISSED** [review]
Fixed the tests and added a retry for timeouts in broken links. 

Approved!

**Review by jyecusch — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**